src/turing_utilities.jl

```diff
@@ -0,0 +1,31 @@
+using Turing
+"""
+    compute_loo(chain::Chain, data, ll_fun)
+
+Computes leave one out (loo) cross validation based on posterior distribution of model parameters.
+
+- `chain`: an MCMCChains object from Turing
+- `model`: a Turing model with data in the form of model(data)
+"""
+function compute_loo(chain::Chains, model)
+    pointwise_lls = pointwise_loglikes(chain, model)
+    psis_output = psis(pointwise_lls)
+    return compute_loo(psis_output, pointwise_lls)
+end
+
+"""
+    pointwise_loglikes(chain::Chains)
```

Comment:
`pointwise_loglikes` seems like it's too similar a name to `pointwise_log_likelihoods`, so they're likely to be easily confused. I'd rename them to either be more distinctive, or else I'd rename these methods to `pointwise_log_likelihoods` as well if that doesn't create any conflicts.

Reply:
I changed it to pointwise_log_likelihoods